Project-I-Social-Card/social-card/src/components/CardComponents/CardContent.js

```diff
@@ -1,2 +1,18 @@
 import React from 'react';
 import './Card.css';
+
+
+const CardContent = props => {
+    return (
+        <>
+        <div className="content-container">
```

Comment:
I would add a secondary className here if you will use content-container also in the header -- that way you can do different styling unique to the bottom, but then have global styles for both the top part of the card and the bottom part of the card. 

something like: ```<div className="content-container bottom">Stuff here</div>```